src/lib/kit/components/Inputs/TimeRangeSelector/utils.ts

```diff
@@ -0,0 +1,15 @@
+export const filterTimeArray = (
+    times: {
+        id: string;
+        content: string;
+        value: string;
+    }[],
+    cutoff: string,
+    direction: 'greater' | 'less',
+) => {
+    if (direction === 'greater') {
+        return times.filter(({value: time}) => time > cutoff);
+    }
+
+    return times.filter(({value: time}) => time < cutoff);
```

Comment:
let's remove the code duplication

just like that
`
return times.filter(({value: time}) => direction === 'greater' ? time > cutoff : time < cutoff)
`